RadixWallet/Features/AssetsFeature/Components/HelperViews/ResourceBalance/ResourceBalanceView.swift

```diff
@@ -465,7 +468,7 @@ extension ResourceBalanceView {
 		}
 
 		private var useSpacer: Bool {
-			amount != nil || fallback != nil || caption1 == nil
```

Comment:
Wasn't a good place for the fix, it did push out completely the text field for amount entry.